app/views/books/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> Highest Rating: <%= @book.reviews.by_rating.last.rating%> </h1>
+<p> <%= @book.reviews.by_rating.last.body %> as reviewed by <%= @book.reviews.by_rating.last.user.name %> </p>
+
+<h2> Lowest Rating: <%= @book.reviews.by_rating.first.rating%> </h2>
+<p> <%= @book.reviews.by_rating.first.body %> as reviewed by  <%= @book.reviews.by_rating.first.user.name %> </p>
```

Comment:
You'd want to create methods  on an instance of Book that you can call in the above lines. You'll want at most to call a single method from your object/class rather than chaining 4 or 5. This would be considered too much logic in your view.